src/components/SelectButton.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { RaisedButton } from 'material-ui';
+import SvgIcon from 'material-ui/SvgIcon';
+
+import * as colors from '../colors';
+
+export const UNSELECTED = 'ADD TO MY COLLECTION';
```

Comment:
So I like actions being cased like this. I don't really like anything else. I also don't think we need to pull this text out into variables. Not sure how others will feel about this and am happy to have the discussion.